src/en/juju-gui-management.md

```diff
@@ -117,7 +110,79 @@ positioned in a way that makes sense to you. These positions are stored in your
 Juju environment, so the next time you open the GUI, things will be as you left
 them.
 
-For an example of this, check out this demonstration of building an OpenStack
-instance in Juju using the GUI:
+## Upgrading Juju GUI
+
+The `upgrade-gui` command downloads the latest published GUI from the streams 
+and replaces the one on the controller. To verify which versions of the GUI 
+are available before the upgrade, try:
+
+```
+juju upgrade-gui --list
+```
+If you want to upgrade (or downgrade) to a specific version of the GUI, 
+provide the revision as a parameter to the upgrade-gui command, where the 
+revision listed by the juju upgrade-gui --list. For example:
```

Comment:
I would probably put juju ugrade-gui --list in backticks so it appears as inline code:
`juju ugrade-gui --list`